Detections/CommonSecurityLog/PaloAlto-NetworkBeaconing.txt

```diff
@@ -0,0 +1,39 @@
+// Name: Detect beaconing based on repetitive time intervals seen in Palo Alto Network traffic Logs
+// Description: This query will identify beaconing patterns from Palo Alto Network traffic logs based on recurrent timedelta patterns. 
+// The query leverags various KQL functions to calculate time delta and then compare it with total events observed in a day to find percentage of beaconing. 
+// results of such beaconing patterns to untrusted public networks should be investigated for any malware callbacks or data exfiltration attempts.
+//
+// Reference Blog about creating dataset to identify network beaconing via repetitive time intervals seen against total traffic between same source-destination pair.
+// http://www.austintaylor.io/detect/beaconing/intrusion/detection/system/command/control/flare/elastic/stack/2017/06/10/detect-beaconing-with-flare-elasticsearch-and-intrusion-detection-systems/
+//
+// Id: f0be259a-34ac-4946-aa15-ca2b115d5feb
+//
+// DataSource: #VMConnection, #CommonSecurityLog
```

Comment:
I don't see the Datasource #VMConnection in this one, I assume this was left over from another query.  Please remove.

Reply:
removed incorrect datasource in the new commit